.github/workflows/update-image-builder.yml

```diff
@@ -0,0 +1,17 @@
+name: update-image-builder
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  update-image-builder:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Push commit to image-builder repo
+        run: |
+          git clone "https://${{ secrets.PROW_BOT_TOKEN }}@github.com/open-cluster-management/image-builder.git"
+          cd image-builder
+          echo "$GITHUB_REPOSITRY@%GITHUB_SHA" > .trigger-openshift-ci"
```

Comment:
Should `$GITHUB_REPOSITRY@%GITHUB_SHA` be `$GITHUB_REPOSITRY@$GITHUB_SHA`?

Reply:
$GITHUB_REPOSITORY :)

Reply:
Oh. Now I see the percent sign in front of GITHUB_SHA. So there were two issues.